services/app-web/src/common-code/domain-models/submission.ts

```diff
@@ -135,6 +136,64 @@ function submissionName(
     return `MCR-${submission.stateCode.toUpperCase()}-${formattedProgramNames}-${padNumber}`
 }
 
+export type RateDataType = {
```

Comment:
Exactly what I was talking about. This is one of the best parts of Typescript is that it's easy to make a little protocol. If we ever change our other types to not fulfill this we'll get errors.